projects/baby_driver/scripts/i2c_write.py

```diff
@@ -0,0 +1,50 @@
+"""Python implementation of the i2c_write function"""
+import can_util
+from message_defs import BabydriverMessageId
+
+
+def i2c_write(port,address, tx_bytes, reg=None):
+    """
+    Writes over I2C to the given port/address
+    Args:
+        port: port of the I2C to write to (1 or 2)
+        address: I2C address to write to (0-255)
+        tx_bytes: list of bytes to write over I2C (a list of ints 0-255)
+        reg: the register to write to, or None if no register is required
+    Raises:
+        Exception: if a non-zero status code was received
+    """
+    if port not in (1,0):
+        raise ValueError("Expected port of 0 (I2C_PORT_1) or 1 (I2C_PORT_2)")
```

Comment:
It's a bit confusing to make users enter 0 for port 1 or 1 for port 2, eh? Better I think would be to take 1 or 2 for port 1 or port 2 respectively, then subtract 1 before packing the data.